engine/src/test/java/io/zeebe/engine/util/WorkflowExecutor.java

```diff
@@ -105,6 +111,20 @@ private void publicMessage(final StepPublishMessage publishMessage) {
         .await();
   }
 
+  private void publishStartMessage(final StepPublishStartMessage publishMessage) {
+    RecordingExporter.messageStartEventSubscriptionRecords(
+            MessageStartEventSubscriptionIntent.OPENED)
+        .withMessageName(publishMessage.getMessageName())
+        .await();
+
+    engineRule
+        .message()
+        .withName(publishMessage.getMessageName())
+        .withCorrelationKey(IntermediateMessageCatchEventBlockBuilder.CORRELATION_KEY_VALUE)
+        .withVariables(publishMessage.getVariables())
+        .publish();
```

Comment:
Currently, we don't have a record for the correlation to the message start event yet. But it will be introduced by my next PR. I suggest adding the check with my PR. Is this okay for you?

Reply:
Sure